ios/PhylloConnectModule.swift

```diff
@@ -14,6 +14,7 @@ import PhylloConnect
 public class PhylloConnectModule: RCTEventEmitter {
     
     var hasObservers:Bool?
+   // var hasBaseUrl = "Test"
```

Comment:
Delete the commented code (unless this is temporary removal and we will need the same code again in future)